### PR TITLE
etcd: Update to version 3.5.11

### PR DESCRIPTION
**Pull request by mzaian**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Update etcd to version 3.5.11 See release: [v3.5.11](https://github.com/etcd-io/etcd/releases/tag/v3.5.11).

**Which issue(s) this PR fixes**:
Fixes NONE #

**Does this PR introduce a user-facing change?**:
NONE

```release-note
etcd: Update to v3.5.11
```

### CODE REVIEW

File: cmd/kubeadm/app/constants/constants.go [hunk 1]
```diff
@@ -464,14 +464,14 @@ var (
...
-		28: "3.5.10-0",
-		29: "3.5.10-0",
+		22: "3.5.11-0",
+		23: "3.5.11-0",
+		24: "3.5.11-0",
+		25: "3.5.11-0",
+		26: "3.5.11-0",
+		27: "3.5.11-0",
+		28: "3.5.11-0",
+		29: "3.5.11-0",
```
Comment (neolit123) [inline cmd/kubeadm/app/constants/constants.go:474]: /cc @SataQiu
maybe we can handle the upgrade in this PR?

**Comment by seans3** [comment]
This PR looks like a duplicate of: https://github.com/kubernetes/kubernetes/pull/122328

That PR is updating 520 files, while this PR is updating 8 files.

**Comment by pacoxu** [comment]
/lgtm
/approve

> This PR looks like a duplicate of: #122328
> 
> That PR is updating 520 files, while this PR is updating 8 files.

#122328 only bump the golang dependencies in go.mod/go.sum and vendor.

This PR bumps the etcd image version as a follow up of https://github.com/kubernetes/kubernetes/pull/122233 and https://github.com/kubernetes/k8s.io/pull/6207.

File: cmd/kubeadm/app/constants/constants.go (hunk 1 above)
Comment (pacoxu) [reply cmd/kubeadm/app/constants/constants.go:474]: Link #122265

**Comment by mzaian** [comment]
Rebased to get the v1.30 change from v1.30 #122265

**Review by SataQiu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**